XCUITests/TranslationSnackBarTest.swift

```diff
@@ -5,15 +5,41 @@
 import XCTest
 
 class TranslationSnackBarTest: BaseTestCase {
-    func testSnackBar() {
+    
+    // This test checks to the correct functionalty of the Translation prompt and Translation is done corrrectly using Google
+    func testSnackBarDisplayed() {
         userState.localeIsExpectedDifferent = true
         navigator.openURL(path(forTestPage: "manifesto-zh-CN.html"))
         waitUntilPageLoad()
-        navigator.goto(TranslatePageMenu)
         waitForExistence(app.buttons["TranslationPrompt.doTranslate"])
+        app.buttons["TranslationPrompt.dontTranslate"].tap()
```

Comment:
Similarly to the action to [translate](https://github.com/mozilla-mobile/firefox-ios/blob/71fd6d0e53ea4fdf72a3bae50ac701f76639bf27/XCUITests/FxScreenGraph.swift#L903) you could create one for don't translate, does it make sense?